public/diagnostic.js

```diff
@@ -1,5 +1,25 @@
 /// Ember Object Diagnostic ///
 
+const Cart = Ember.Object.extend({
+  orders: [],
+  addToCart: function(unitPrice, item, quantity){
+    Cart.orders.push(Order.create({unitPrice: unitPrice, item: item, quantity: quantity }));
+  },
```

Comment:
If the surrounding code supported it, then this `addToCart` method wouldn't error, but it isn't doing what the instructions asked.  We need `addToCart` to add each Item to the array.